labels/combined_windows.json

```diff
@@ -88,12 +88,7 @@
             "2014-04-15 17:34:00.000000"
         ]
     ],
-    "realAWSCloudwatch/ec2_cpu_utilization_c6585a.csv": [
-        [
-            "2014-04-14 10:34:00.000000",
-            "2014-04-15 20:04:00.000000"
-        ]
-    ],
+    "realAWSCloudwatch/ec2_cpu_utilization_c6585a.csv": [],
```

Comment:
It never should've been a truth label, but was included b/c of the bug fixed in commit 49c9aaa. The number of raw label files with an entry for a given data file is used as the number of labelers for that data file. "realAWSCloudwatch/ec2_cpu_utilization_c6585a.csv" is in three of the raw label files, so the algorithm should count three total labelers for this data file. However, only one of the three labelers have an anomaly labeled for this data file, so w/ the old logic the label combining algorithm would count one total labeler for this file. After the bug fix this anomaly label is counted once out of three total labelers, and thus does not pass the threshold.